app/services/duplicate_application.rb

```diff
@@ -70,8 +74,7 @@ def duplicate
         change_references_to_not_requested_yet(references_cancelled_at_eoc)
       end
 
-      references_completed = apply_again? && new_application_form.complete_references_information?
-      new_application_form.update!(references_completed:)
+      new_application_form.update!(references_completed: false)
```

Comment:
Is this related to the HESA work?

Reply:
Actually no. We don't have the concept of apply_2 anymore and no one is calling apply_2 (a.k.a apply_again?) so this will always be false. But I can't undo this because it is not related to HESA.

Reply:
Wdyt?

Reply:
I don't feel strongly about it. Just wasn't sure if I was missing something.